Source/WebCore/css/process-css-pseudo-selectors.py

```diff
@@ -741,6 +795,7 @@ def main():
     input_data = json.load(input_file)
     webcore_defines = [i.strip() for i in sys.argv[-1].split(' ')]
 
+    InputValidator(input_data)
```

Comment:
Arguably the style checker should run this, but not sure how involved that is.

Reply:
This will fail at build time like it does for CSSProperties.json. I think it's slightly nicer that way, and it's also easier that to have validation implemented in the same file as that code generation.